src/style.css

```diff
@@ -1,3 +1,96 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media all and (max-width: 1024px) {
+  .header {
+    width: 1024px;
+  }
+}
+
+@media all and (min-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
```

Comment:
Actually media is not required in this task, you can simply add min-width to your header and thats it